packages/console/src/pages/AuditLogDetails/index.tsx

```diff
@@ -22,19 +23,23 @@ import EventIcon from './components/EventIcon';
 import * as styles from './index.module.scss';
 
 const AuditLogDetails = () => {
-  const { logId } = useParams();
+  const { id: userId, logId } = useParams();
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { data, error } = useSWR<LogDTO, RequestError>(logId && `/api/logs/${logId}`);
+  const { data: userData } = useSWR<User, RequestError>(userId && `/api/users/${userId}`);
+
   const isLoading = !data && !error;
 
+  const backLink = userId ? `/users/${userId}` : '/audit-logs';
```

Comment:
just maybe: better determine the backlink according to the current path instead of its path parameter like `userId`?

Reply:
If this page comes from the `user-details-logs`, it will contain the `userId`, and we will use the `userId` later.
I thought it's good to use `userId` on this page.